CI.jenkinsfile

```diff
@@ -0,0 +1,54 @@
+// Norma CI test norma using "make test"
+
+pipeline {
+	agent { label 'norma' }
+	
+	options {
+		timestamps ()
+		timeout(time: 3, unit: 'HOURS')
+	}
+
+	parameters {
+		string(defaultValue: "main", description: 'Can be either branch name or commit hash.', name: 'NormaVersion')
+	}
+
+	environment {
+		GOROOT = '/usr/local/go'
+	}
+	
+	stages {
+		stage('Clone Norma') {
+			steps {
+				script {
+					currentBuild.description = 'Building on ${env.NODE_NAME}'
+                		}
+				checkout scmGit(
+					branches: [[name: '${NormaVersion}']],
+					userRemoteConfigs: [[url: 'https://github.com/Fantom-foundation/Norma.git']]
+				)
```

Comment:
Is it necessary to clone norma repository here?

Reply:
Fresh Norma installation is intentional. This is to avoid any residual Docker Image created by previous version of sonic, should it is baked into the GCP image.

Reply:
Sorry I missed the point of the comment. This `CI.jenkinsfile` is meant for the manual triggering targeting specific branch. There's a separate `PR.jenkinsfile` that does not have this because it knows the branch from the specific PR.